tests/manage/pv_services/test_resource_deletion_during_pod_pvc_deletion.py

```diff
@@ -114,7 +114,14 @@ def disruptive_base(
         # Start IO on each pod
         log.info("Starting IO on pods")
         for pod_obj in self.pod_objs:
-            pod_obj.run_io(storage_type='fs', size=f'{self.pvc_size - 1}G')
+            if pod_obj.pvc.access_mode == constants.ACCESS_MODE_RWX:
```

Comment:
Please add a comment in the code for what's being done here.
Why are we filling half of the disk in case of RWX?

Reply:
Comment added.
There will be two pods which use same RWX PVC